server/public/templates/admin/settings.html

```diff
@@ -37,25 +37,41 @@ <h5>Global</h5>
             </div>
 
             <h5>Database</h5>
-            <fieldset disabled>
             <div class="form-group">
               <label for="database.type" class="col-xs-4 col-sm-3 control-label">Type</label>
               <div class="col-xs-8 col-sm-9">
-                <select name="database.type" ng-model="settings.database.type" class="form-control" readonly>
+                <select name="database.type" ng-model="settings.database.type" class="form-control">
                   <option value="sqlite">SQLite</option>
-                  <option value="mongo">MongoDB</option>
-                  <option value="dynamo">DynamoDB</option>
+                  <option value="postgress">Postgress</option>
```

Comment:
Minor niggle ;) s/Postgress/Postgres/

Reply:
Oopsie can't say I've ever used postgres ... Guess my brain autocorrected it lol.